config/onprem-iso-config.ign

```diff
@@ -29,7 +29,7 @@
         "path": "/etc/assisted-service/environment",
         "contents": {
           "compression": "gzip",
-          "source": "data:;base64,H4sIAAAAAAAC/7yVXY+jNhSG7/kZvgbzEQYYJC6YxJ0gZQMLJKtVVVkecBJrAVObpDsd5b9Xztdk2my72mr2AgmfA+9j/ByJLC3KxxwVH2d4EpfxQ1ygiHVyIE1DhXbVzeKi+JTmk4jULeuuO4sC5afq5AFP06KMbMeHFrSgrSpZmpfRnTty1OLtwyr0dTWPP1zDC5QvkzHCak94kc+izTD0MjRNIiWTA60N0jPY8Io0kPe0kxu2GiDjYeC6I22Csln6GZdx/ojKiHe9oK1WlGkeP6JoxRoqn+WgSifKOMZjlJc4i8tpZNZkIGa3Zt1XQ9JmJdm6ozWsxKClGZoX0+SXEi9RXiTpvIhegAs9EL6Amsm+Ic+4Iy0FoapC2wM6ELShRFK8o0Iy3t1qsZas1Tu/b8kzZNy8fJBxesKo6c7kVX9eh8cI42vgYc9VSZuKy0vO+axaJgQXV+dT8dbst09XgJ1r1rSnXU27ilFpHoJMF3rqgsFxbRzuTzSjYTsKj/eQSX6hC86HlXwPvCIax/gLuF1fwFcn68HAgY7lWLZj3dmBYxkW0IHc9j0XA27ojjYgBL3g9bYa1Dt7HbjQv+nPh/ff0Pe28yP2fHj/jvJ8dUH/Is+H/s+U9xb//fJ8GIyUPNsaOXe25/6nPB3UdEW2zQDCQWzpwWVw02UALWNVwdE3hN5o/4jVc8y7qe0FPePMV9zF89/4P0H2v27o+80HMHCP5u8sz7Gd0U3zT3QgYL/XssXDLBnjcTov42SOcpyjx6Qo8wQV0UmXlmRLDxeL7PD/UcOhxYtyisvPGYo63lFt+gkv41kyictUBXxcJDn6gOZlEf36Al63dh4wHbREDlSo6ar6La64oBKErg4EaXHLnkBoe6PA1dXsfcGS/UnxWhUdSwen3xpRQ4uP/Z7SGg8bQeWGNzVuJQhta6+DP7j48g+Kc0UJ7Hvn/0H2v2l/BQAA///FI7r4+QcAAA=="
+          "source": "data:;base64,H4sIAAAAAAAC/7yVXW+jOBSG7/kZvgbzEQoUiQuaeBukTGCAdDRarSwXnMQawKxNstOt8t9XzlfT2czuaEbtBRI+B97H+DkSWVqU9zkqPs7wJC7ju7hAEevkQJqGCu2im8VF8SnNJxGpW9ZddhYFyo/VyR2epkUZ2Y4PLWhBW1WyNC+jG3fkqMXrh1Xoy2oef7iEFyh/SMYIqz3hRT6L1sPQy9A0iZRMDrQ2SM9gwyvSQN7TTq7ZcoCMh4HrjrQJymbpZ1zG+T0qI971grZaUaZ5fI+iJWuofJKDKh0p4xiPUV7iLC6nkVmTgZjdinVfDUmbpWSrjtawEoOWZmheTJPfSvyA8iJJ50X0DFzogfAZ1Ez2DXnCHWkpCFUV2h7QgaANJZLiLRWS8e5ai7Vkpd75c0OeIOPm+YOM4xNGTbcmr/rTOjxEGF8DD3uuSlpXXJ5zTmfVMiG4uDifirdmv3m8AGxds6Y97WraVYxKcx9kutBTFwwOa2N/f6QZDdtSeLiHTPIzXXA+LOVb4BXROMSfwe3qDL44WQ8GDnQsx7Id68YOHMuwgA7kpu+5GHBDt7QBIegFrzfVoN7Z6cCF/lV/Prz9jr7XnZ+x58PbN5Tnqwv6Z3k+9N9T3mv8j8vzYTBS8mxr5NzYnvu/8nRQ0yXZNAMIB7Ghe5fBVZcBtIxlBUffEXql/TNWTzFvprYX9IQzX3Bnz9/w30H2f27ox80HMHAP5m8sz7Gd0VXzj3QgYLfTssXdLBnjcTov42SOcpyj+6Qo8wQV0VGXlmQPHi4W2f7/o4ZDixflFJefMxR1vKPa9BN+iGfJJC5TFfBxkeToA5qXRfT7M3jZ2mnAdNASOVChpqvqN7jigkoQujoQpMUtewSh7Y0CV1ez9wVL9jfFK1V0LB0cf2tEDS0+9HtKazysBZVr3tS4lSC0rZ0O/uLiy78ozgUlsG+dX4XIjn9DCC4II8f3gl9D7P7Q/gkAAP//Uohy11wIAAA="
```

Comment:
@YuviGold any reason we have compression (gzip) here?
without it, the data will be more readable.

Reply:
No idea.
@rwsu ?